frontend/app/src/store/defi/getters.ts

```diff
@@ -1343,16 +1346,13 @@ export const getters: Getters<DefiState, DefiGetters, RotkehlchenState, any> = {
   },
 
   yearnVaultsProfit:
-    (
-      { yearnVaultsHistory, yearnVaultsV2History },
-      _dg,
-      _rs,
-      { 'balances/assetSymbol': assetSymbol }
-    ) =>
+    ({ yearnVaultsHistory, yearnVaultsV2History }, _dg, _rs) =>
```

Comment:
same here with `_dg` `_rs`

Reply:
Basically all dg rs in this file should be removed, I also forgot to refactor `getIdentifierForSymbol` in this file